modules/sov-ibc-transfer/src/context.rs

```diff
@@ -336,6 +336,12 @@ impl<'ws, S: Spec> TokenTransferExecutionContext for IbcTransferContext<'ws, S>
                     let minter_address = account.address.clone();
                     // Only the transfer module is allowed to mint
                     let authorized_minters = vec![self.ibc_transfer.address.clone()];
+                    // Make sure to use `ibc_transfer` address as the sender
```

Comment:
`None` branch seems very important. Can we refactor it to be a method for `IbcTransferContext`?

Reply:
[b927153](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/b9271531b1b259510603b1c0048725a82dceea78)